R/PLNnetwork.R

```diff
@@ -72,18 +73,25 @@ PLNnetwork <- function(formula, data, subset, weights, penalties = NULL, control
 #' @seealso [PLN_param()]
 #' @export
 PLNnetwork_param <- function(
-    backend           = "nlopt",
+    backend           = c("nlopt", "torch"),
+    covariance        = c("fixed", "spherical", "diagonal"),
```

Comment:
I'm mixed about this: to avoid potential problems, it would be better to allow only `covariance = "fixed"` (especially since f801ca6) reverts back to `PLNnetworkfit <-R6Class(inherits = PLNfit_fixedcov)`

Reply:
@maddyduran and I discussed this and we think the use case for covariance = "spherical" or "diagonal" in PLNnetwork is when you have fewer samples than you have species. IIRC, the issue was the call to solve() in "fixed" optimize() function.